service/api_logic/filter_manager/filter_manager_factory.py

```diff
@@ -0,0 +1,26 @@
+from service.api_logic.filter_manager.games_filter_manager import GamesFilterManager
+from service.api_logic.filter_manager.league_filter_manager import LeagueFilterManager
+from service.api_logic.filter_manager.news_filter_manager import NewsFilterManager
+from service.api_logic.filter_manager.teams_filter_manager import TeamsFilterManager
+from sqlalchemy.orm import Query
+from exept.exeptions import IncorrectModelFromFilterManager
+
+class FilterManagerFactory:
```

Comment:
It's named a `factory` but it does not actually return any instances of created objects. This has to be renamed somehow, or removed.